GUI/FileSpecificParamWindow.xaml.cs

```diff
@@ -0,0 +1,126 @@
+﻿using EngineLayer;
+using MzLibUtil;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using TaskLayer;
+
+namespace MetaMorpheusGUI
+{
+    /// <summary>
+    /// Interaction logic for FileSpecificParamWindow.xaml
+    /// </summary>
+    /// 
+
+
+    public partial class FileSpecificParamWindow : Window
+    {
+        internal FileSpecificSettings[] FileSpecificSettingsList { get; private set; }
+        internal ObservableCollection<RawDataForDataGrid> SelectedRaw { get; private set; }
+
+        public FileSpecificParamWindow(ObservableCollection<RawDataForDataGrid> selectedRaw)
+        {
+            InitializeComponent();
+            SelectedRaw = selectedRaw;
+            FileSpecificSettingsList = new FileSpecificSettings[selectedRaw.Count];
+
+            foreach (Protease protease in GlobalEngineLevelSettings.ProteaseDictionary.Values)
+                proteaseComboBox.Items.Add(protease);
+            proteaseComboBox.SelectedIndex = 12;
+
+            foreach (string initiatior_methionine_behavior in Enum.GetNames(typeof(InitiatorMethionineBehavior)))
+                initiatorMethionineBehaviorComboBox.Items.Add(initiatior_methionine_behavior);
+
+            productMassToleranceComboBox.Items.Add("Absolute");
+            productMassToleranceComboBox.Items.Add("Ppm");
+
+        }
```

Comment:
Add a method to populate textboxes/checkboxes etc with file specific params corresponding to the selected raw files

Reply:
See e.g. SearchTaskGUI.cs